easybuild/easyconfigs/s/SCons/SCons-3.0.1-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'SCons'
+version = '3.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.scons.org/'
+description = "SCons is a software construction tool."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['24475e38d39c19683bc88054524df018fe6949d70fbd4c69e298d39a0269f173']
+
+dependencies = [('Python', '3.6.3')]
```

Comment:
Previous easyconfigs didn't need a Python package (the one from the OS was used). Maybe comment here why this change? I suppose SCons 3 needs Python 3

Reply:
We have one version that relies on the system Python, because we wanted it to use in combination with Python 3 while `SCons` 2.x doesn't actually support Python 3 yet (so we called `scons` using the `python2` from the OS).

This was a bit hackish, and with `SCons` 3.x that is no longer needed, since they now properly support both Python 2.7 and 3.x, so no need to rely on the system Python anymore.